doc/devel/development_setup.rst

```diff
@@ -127,6 +127,8 @@ The simplest way to do this is to use either Python's virtual environment
         <file folder location>\Scripts\activate.bat  # Windows cmd.exe
         <file folder location>\Scripts\Activate.ps1  # Windows PowerShell
 
+      On Mac, you may need to type python3 instead of python.
```

Comment:
```suggestion
      On some systems, you may need to type `python3` instead of `python`
      (see [PEP-394](https://peps.python.org/pep-0394)).
```